prow/plugins/dco/dco.go

```diff
@@ -81,8 +98,31 @@ func helpProvider(config *plugins.Configuration, enabledRepos []string) (*plugin
 	return pluginHelp, nil
 }
 
+// optionsForRepo gets the plugins.Dco struct that is applicable to the indicated repo.
+func optionsForRepo(config *plugins.Configuration, org, repo string) *plugins.Dco {
+	fullName := fmt.Sprintf("%s/%s", org, repo)
+	for i := range config.Dco {
+		if !strInSlice(org, config.Dco[i].Repos) && !strInSlice(fullName, config.Dco[i].Repos) {
+			continue
+		}
+		return &config.Dco[i]
+	}
+	return &plugins.Dco{}
+}
+
+// strInSlice returns true if any string in slice matches str exactly
+func strInSlice(str string, slice []string) bool {
```

Comment:
You could just use one loop above and not need to factor this out -- I think that may be more readable, otherwise we would accumulate a lot of these. Alternatively:

```go
for _, repoConfig := range config.Dco {
    if !sets.NewString(repoConfig.Repos...).HasAny(org, fullName) {
        // etc
    }
}
```

Reply:
I've experimented and used similar snippet in the `DcoFor` function. The unit tests are passing, but let me know if it's okay to use it.